apps/nextjs/src/app/aila/help/index.tsx

```diff
@@ -187,7 +190,7 @@ const Help = () => {
               each section, for example:
             </p>
             <ul className="mb-6 list-disc pl-12">
-              <li>adapt the content for a specific reading age.</li>
+              <li>increase or decrease the literacy level of your keywords</li>
```

Comment:
This addition doesn't have a full stop at the end of it whilst all other items in this list do. I'm sure the general rule is that items in lists should not have full stops, so it might be that the other items are wrong, not this one.